executor/metric_reader.go

```diff
@@ -134,15 +135,15 @@ func (e *MetricRetriever) genRows(value pmodel.Value, r promQLQueryRange, quanti
 		matrix := value.(pmodel.Matrix)
 		for _, m := range matrix {
 			for _, v := range m.Values {
-				record := e.genRecord(m.Metric, v, r, quantile)
+				record := e.genRecord(m.Metric, v, quantile)
```

Comment:
Seems the parameter `r` at L131 can be eliminated.

Reply:
great,done.